src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -16,6 +16,8 @@
     <!-- ToDo: Remove this P2P reference once new packages are produced and updated -->
     <ProjectReference Include="..\..\System.Net.Requests\ref\System.Net.Requests.csproj">
     </ProjectReference>
+    <!-- ToDo: Remove this P2P reference once new packages are produced and updated -->
```

Comment:
We really need to look at breaking this CodeDom dependency, otherwise we will have a netstandard2.0 set of APIs in XML that depend on CodeDom which isn't part of netstandard2.0